src/tests/class_tests/openms/CMakeLists.txt

```diff
@@ -91,7 +91,10 @@ target_link_libraries(Base64_test ZLIB::ZLIB)
 target_link_libraries(LPWrapper_test CoinOR::CoinOR)
 target_link_libraries(SpectraSTSimilarityScore_test Eigen3::Eigen)
 target_link_libraries(BinnedSpectrum_test Eigen3::Eigen)
-target_link_libraries(HDF5_test HDF5::HDF5)
+
+if (WITH_HDF5)
+  target_link_libraries(HDF5_test HDF5::HDF5)
```

Comment:
Maybe the same needs to be done for the OpenMS lib?

Reply:
yeah I will add it...